.eslintrc.cjs

```diff
@@ -45,6 +45,7 @@ module.exports = {
     '@typescript-eslint/no-empty-interface': 'off',
     '@typescript-eslint/ban-ts-comment': 'off',
     '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/no-duplicate-type-constituents': 'off',
```

Comment:
For future us, we did this because eslint's fix on save was eliminating a redundant arm from this type union (which we're no longer using) which we wanted to keep.

```
type T = SledPolicy['kind'] | PhysicalDiskPolicy['kind']
```